_includes/_footer.htm

```diff
@@ -142,6 +142,9 @@ <h6>Main Guides</h6>
                     <li>
                         <a href="/assets/docs/grc-bluepaper-section-1.pdf">Gridcoin Staking Bluepaper</a>
                     </li>
+                    <li>
+                        <a href="/assets/docs/scraper-summary.pdf">Executive Scraper Summary</a>

```

Comment:
Might we also want to link this to the scraper wiki page?

Reply:
This is under the publications header. The wiki article is informal imo

Reply:
I think it is ok.